organization/planning/plan-of-work-2021.md

```diff
@@ -0,0 +1,225 @@
+---
+title: HSF Plans for 2021
+author: Graeme Stewart
+layout: plain_toc
+---
+
+## Preamble
+
+*In the first few meetings of 2021 the HSF established some high-level
```

Comment:
```suggestion
* In the first few meetings of 2021 the HSF established some high-level
```

Reply:
Suggesting the trivial extra space as otherwise I believe MD will not itemize properly.